packages/components/src/experience-control/index.stories.tsx

```diff
@@ -0,0 +1,46 @@
+import { Meta, StoryObj } from '@storybook/react';
+import { ExperienceControl } from './index';
+
+const meta: Meta< typeof ExperienceControl > = {
+	title: 'Components/ExperienceControl',
+	component: ExperienceControl,
+};
+
+export default meta;
+type Story = StoryObj< typeof ExperienceControl >;
+
+export const Default: Story = {
+	args: {
+		label: 'How was your experience?',
+		selectedExperience: 'good',
+		onChange: () => {},
```

Comment:
Instead of a noop like this, the convention is to [add](https://storybook.js.org/docs/essentials/actions#via-storybooktest-fn-spy-function) `args: { onClick: fn() }` in the `meta` so they can be logged to the Actions panel.

Reply:
Fixed it here: https://github.com/Automattic/wp-calypso/pull/103000